pkg/elemental/elemental.go

```diff
@@ -659,19 +670,20 @@ func CopyCloudConfig(c types.Config, path string, cloudInit []string) (err error
 }
 
 // SelinuxRelabel will relabel the system if it finds the binary and the context
-func SelinuxRelabel(c types.Config, rootDir string) error {
-	policyFile, err := utils.FindFile(c.Fs, rootDir, filepath.Join(cnst.SELinuxTargetedPolicyPath, "policy.*"))
+func SelinuxRelabel(c types.Config, rootDir string, extraPaths ...string) error {
 	contextFile := filepath.Join(rootDir, cnst.SELinuxTargetedContextFile)
 	contextExists, _ := utils.Exists(c.Fs, contextFile)
 
-	if err == nil && contextExists && c.Runner.CommandExists("setfiles") {
+	if contextExists && c.Runner.CommandExists("setfiles") {
```

Comment:
maybe we should check that rootDir is RW as well? might be enough to get the recovery test green.

Reply:
It is RW... disable selinux and  the test will pass, I believe the snapper error msg is just misleading, it could not write due to some denial, but not because of being RO.